### PR TITLE
first

### PR DESCRIPTION
**Pull request by voloshynamaryna11**

### CODE REVIEW

File: src/main/java/internet/market/controllers/AddProductToShoppingCartController.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 public class AddProductToShoppingCartController extends HttpServlet {
-    private static final Long USER_ID = 1L;
+    private static final String USER_ID = "user_Id";
```
Comment (AnaLanda) [inline src/main/java/internet/market/controllers/AddProductToShoppingCartController.java:13]: Just a remark) The task in HW said we need to 'remove all static variables `user_id`", so I'm not sure if we still need to keep the constant variable `USER_ID` or just use `user_id` as a parameter in `getAttribute()`. Both ways work

**Review by AnaLanda — COMMENTED** [review]
Looks good! :smiley:

File: src/main/java/internet/market/controllers/AddProductToShoppingCartController.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/main/java/internet/market/controllers/AddProductToShoppingCartController.java:13]: @AnaLanda yes, both ways work)

File: src/main/java/internet/market/controllers/AddProductToShoppingCartController.java [hunk 2]
```diff
@@ -23,7 +23,8 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         String productId = req.getParameter("id");
         Long id = Long.valueOf(productId);
         shoppingCartService.addProduct(shoppingCartService
-                .getByUserId(USER_ID), productService.get(id));
+                .getByUserId((Long) req.getSession().getAttribute(USER_ID)),
```
Comment (KseniiaMakarova) [inline src/main/java/internet/market/controllers/AddProductToShoppingCartController.java:26]: I would recommend to use local variables sometimes to make the code more readable

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**